keyboards/cannonkeys/atlas/rules.mk

```
@@ -0,0 +1,16 @@
+# MCU Name
+MCU = STM32F072
+
+# Build Options
+#   comment out to disable the options.
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
```

No caps lock code, so this wont actually do anything
```suggestion
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```